dockerfiles/ubuntu/Dockerfile

```diff
@@ -0,0 +1,195 @@
+FROM ubuntu:bionic AS resource
+
+
+RUN apt-get update && apt-get install -y \
+  curl \
+  git \
+  gnupg \
+  gzip \
+  jq \
+  openssl \
+  libssl-dev \
+  make \
+  g++ \
+  openssh-client \
+  libstdc++6 \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
+RUN apt-get install -y git-lfs
```

Comment:
yes, since we are using a shell script to get the package repo of `git-lfs` as it is not included in the official ubuntu repo.

Reply:
`git-lfs` is hosted on it's own package repo that the line 18 sets up (yeah it's kinda iffy we're running some arbitrary bash script from the internet but this is the "official" way).

It's separated because we need to install curl to download the setup script